### PR TITLE
Tests Yauheni

### PR DESCRIPTION
**Pull request by evgkas**

### CODE REVIEW

**Review by khayitmuradov — APPROVED** [review]

**Review by sirojbekdev — APPROVED** [review]

File: UI/Pages/LoginPage.cs [hunk 1]
```diff
@@ -10,19 +10,19 @@ public class LoginPage :AbstractPage
         private static readonly By rememberMeCheckbox = By.CssSelector("#wpRemember");
         private static readonly By loginButton = By.CssSelector("#wpLoginAttempt");       
 
-        public void EnterUsername(string username, int timeSeconds)
+        public void EnterUsername(string username)
         {
-            WebDriverExtension.SendKeysToElement(usernameField, username, timeSeconds);
+            WebDriverExtension.SendKeysToElement(usernameField, username);
```
Comment (NikitaGavruk) [inline UI/Pages/LoginPage.cs:15]: Should return new LoginPage(). Same in functions below

File: UI/Steps/LoginPagesSteps.cs [hunk 2]
```diff
@@ -11,11 +11,11 @@ public class LoginPagesSteps
     {
         private static LoginPage loginPage = new LoginPage();
 
-        public static MainPage Login(string username, string password, int timeSeconds)
+        public static MainPage Login(string username, string password)
         {
-            loginPage.EnterUsername(username, timeSeconds);
-            loginPage.EnterPassword(password, timeSeconds);
-            loginPage.ClickToLoginButton(timeSeconds);
+            loginPage.EnterUsername(username);
```
Comment (NikitaGavruk) [inline UI/Steps/LoginPagesSteps.cs]: As this page functions return Page then you can made it via
loginPage.EnterUsername(username).EnterPassword(password).ClickToLoginButton()
Same in functions below

File: UI/Tests/Test.cs [hunk 3]
```diff
@@ -1,9 +1,57 @@
...
+            string actualTitle = MainPageSteps.Search(expectedTitle)
+            .GetTitle();
+            CustomLogger.LogInfo(Utils.LogLevel.Info, "Verify that loaded right article page");
+            Assert.That(expectedTitle, Is.EqualTo(actualTitle));
+        }
+
+        [Test]
+        public void CheckArticleEditHistory()
+        {
+            CustomLogger.LogInfo(Utils.LogLevel.Info, "Go to Main page");
```
Comment (NikitaGavruk) [inline UI/Tests/Test.cs]: Delete it or move after login

File: UI/Tests/Test.cs [hunk 4]
```diff
@@ -1,9 +1,57 @@
...
+using UI.Steps;
 
 namespace UI.Tests
 {
 	[TestFixture]
 	public class Test : BaseTest
 	{
-	}
+        private static MainPage mainPage = new MainPage();
+        private static string username = "AliceTestAcc4587";
```
Comment (NikitaGavruk) [inline UI/Tests/Test.cs]: Let's create a new xml(or another extension) file in Core project which will contain this data and name it TestsConfig.xml

**Review by NikitaGavruk — CHANGES_REQUESTED** [review]

**Comment by evgkas** [comment]
I have added TestDataReader to read user credentials from json. This class may be extended further.

**Comment by NikitaGavruk** [comment]
@evgkas please fix conflict and then merge it to main

**Pull request merged**